packages/frontend/src/components/AudioRecorder/MicRecorder.ts

```diff
@@ -0,0 +1,180 @@
+/**
+ * This file is based on https://github.com/closeio/mic-recorder-to-mp3/blob/master/src/mic-recorder.js
+ *
+ * @license MIT Copyright (c) 2017 Elastic Inc. (Close.io) see ./LICENSE
```

Comment:
The LICENSE file is empty, I assume it's not intentional?

Reply:
Right, it wasn't